pystiche_papers/johnson_alahi_li_2016/_loss.py

```diff
@@ -38,6 +38,25 @@ def content_loss(
     layer: str = "relu2_2",
     score_weight: Optional[float] = None,
 ) -> ops.FeatureReconstructionOperator:
+    r"""Content_loss from "Perceptual Losses for Real-Time Style Transfer and
+    Super-Resolution" by Johnson, Alahi, and Li (original authors) :cite:`JAL2016`.
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see FIXME.
```

Comment:
Where do we want to put these details? We could either gather them in one point (for example above the API in the docs) or within each docstring. Thoughts?

Reply:
I think above the API is the better choice.

Reply:
This was my first idea, but I disagree with it now. I propose we put a summary ("These functions include changes compared to the paper") on top of the API, but describe the actual changes within the docstring. I wouldn't do this in the `Args` section, but rather below it. Could you make a suggestion?